Tao_Current.txt

```diff
@@ -78,83 +89,96 @@ After the Internet Society (ISOC) was formed in January 1992, the IAB proposed t
 
 The IETF met in Amsterdam, The Netherlands, in July 1993. This was the first IETF meeting held in Europe, and the US/non-US attendee split was nearly 50/50. The IETF first met in Asia (in Adelaide, Australia) in 2000.
 
-Currently, the IETF meets in North America, Europe, and Asia. The intention is to meet once a year in each region, although due to scheduling issues there are often more meetings in North America and fewer in Asia and Europe. The number of non-US attendees continues to be high — about 50%, even at meetings held in the United States.
+Currently, the IETF meets in North America, Europe, and Asia. We have also met in Latin America, but this is currently not part of the rotation schedule. The intention is to meet once a year in North America, Europe, and Asia, although due to scheduling and logistical issues this might vary. The number of non-US attendees continues to be high — about 60%, even at meetings held in the United States ([https://datatracker.ietf.org/stats/meeting/continent/]).
+
```

Comment:
This is a stronger statement than in https://tools.ietf.org/html/draft-ietf-mtgvenue-meeting-policy-07#section-2, which says: " It is important to note that such rotation and any
   effects to distributing travel pain should be considered from a long-
   term perspective.  While a potential cycle in an IETF year may be a
   meeting in North America in March, a meeting in Europe in July, and a
   meeting in Asia on November, the 1-1-1 policy does not imply such a
   cycle, as long as the distribution to these regions over multiple
   years is roughly equal."